_security-plugin/multi-tenancy/mt-agg-view.md

```diff
@@ -0,0 +1,79 @@
+
+Once you identify a saved object of interest, you can then switch to that tenant to work with the object.
+
+To access saved objects, expand the top menu and select **Management > Stack Management > Saved Objects**. The Saved Objects window opens. By default, all tenants the user has permissions for are displayed along with all saved objects associated with the tenants.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/agg-view-saved-objects.png" alt="Dashboards Saved Objects view with tenant object aggregation" width="700">
+
+As an experimental feature, aggregate view for saved objects is kept behind a feature flag and must be enabled in the `opensearch_dashboards.yml` file before the feature is made available. See [Enabling aggregate view](#enabling-aggregate-view-for-saved-objects) for more information.
+{: .note }
+
+### Feature benefits
+
+- Implementing an aggregate view for all saved objects on one screen allows you to quickly locate an object of interest and determine which tenant is associated with it. Once you locate an object, you can select the appropriate tenant and work with the object.
+- This feature also adds a Tenant column to the Saved Objects table, which allows you to filter the view by tenants and their associated saved objects.
+
+### Plans for future development
+
+In subsequent releases, we plan to expand the functionality of this feature to include the ability to perform actions directly from aggregate view and share items without having to first select a specific tenant. In the longer term, OpenSearch plans to evolve multi-tenancy so that it becomes a much more flexible tool for sharing objects among users and employs a more sophisticated way of assigning the roles and permissions that facilitate sharing. To learn more about the features being proposed for future releases, see the GitHub issue [Dashboards object sharing](https://github.com/opensearch-project/security/issues/1869).
+
+### Known limitations
+
+In this first experimental phase of development, there are some limitations that should be observed before enabling the feature and using it in a test environment:
+
+* The feature can only be used in a new cluster. At this time, the feature is not suported by clusters already in use.
+* Also, the feature should be used only in a test environment, not in production. 
+* Finally, once the feature has been enabled and used in a test cluster, the feature cannot be disabled for the cluster. Disabling the feature once it has been used to work with tenants and saved objects can result in the loss of saved objects and have an impact on tenant-to-tenant functionality. 
```

Comment:
for conversational tone, use common contractions like "can't"

Reply:
Either "cannot" or "can't" is fine.

Reply:
oh, ok. I think we addressed this before, and we were going to update the style guide accordingly.

Reply:
Right. Thus either is fine.